src/cap-description.adoc

```diff
@@ -124,6 +138,37 @@ If a capability grants <<r_perm>> and <<c_perm>>, but no <<lm_perm>>, then a cap
 The rules specified by <<ACPERM>> are followed when <<w_perm>> and <<lm_perm>> are removed, so additional permissions may also be removed.
 Clearing a capability's <<lm_perm>> and <<w_perm>> allows sharing a read-only version of a data structure (e.g. a tree or linked list) without making a copy.
 
+[#sl_perm,reftext="SL-permission"]
+Store Level Permission (SL):: This permission allows limiting the propagation of capabilities using the following logic: capabilities with a <<section_cap_level>> less than the inverse of <<sl_perm>> will be stored with the tag cleared.
+With one bit for SL and CL each, this permission functions as follows:
+// This awkward wording above allows a multi-bit scheme:
+// For a multi-bit SL and L this becomes the following scheme that allows capabilities as low as level = ~store_level (and anything "more global").
+// In a one bit scheme:
+// 1 -> allow storing as low as level ~1 = 0 -> level 0 (local) and 1 (global) is okay
+// 0 -> allow storing as low as level ~0 = 1 -> only level 1 (global)
+// In a two bit scheme:
+// 11 -> allow storing as low as level ~11 = 00 -> levels 0-3 are ok
+// 10 -> allow storing as low as level ~10 = 01 -> level 1, 2 and 3 are ok
+// 01 -> allow storing as low as level ~01 = 10 -> level 2 and 3 are ok
+// 00 -> allow storing as low as level ~00 = 11 -> only level 3 is ok
+- If this permission (as well as <<c_perm>> and <<w_perm>>) is set then capabilities may be stored via this capability regardless of their associated <<section_cap_level>>.
+- If the permission is not set, then any capability with a <<section_cap_level>> of zero (i.e. _local_), will be stored with the tag cleared.
+
+NOTE: Future extensions may widen this permission and <<section_cap_level>> to more than one bit.
+
+[#ll_perm,reftext="LL-permission"]
+Load Level Permission (LL):: If this permission is not set then any tagged capabilities loaded via this capability will have the <<ll_perm>> cleared and the <<section_cap_level>> restricted to the <<section_cap_level>> of the authorizing capability.
+
+Thus, if LG and GL are cleared before delegating a capability then it, and any capability loaded via it (including via indirection),
+may be stored only via capabilities with PERMIT_STORE_LOCAL_CAPABILITY. This
+limits the ability of the delegee to retain capabilities to a delegated data structure or
+part thereof, making it easier to later revoke access to the delegated data structure. In
+the case of loaded sealed capabilities, GL, but not LG, is cleared, making this an exception to the immutability of sealed capabilities. Thus, a sealed capability reached
+via an authority lacking LG may be stored only through a PERMIT_STORE_LOCAL_-
+CAPABILITY authority, as may its unsealed form, but the further authority borne by
+the sealed capability once unsealed is unaltered. This differs from the behavior of
+LM on sealed capabilities. Untagged capabilities are unaffected.
```

Comment:
Was this paragraph taken from a paper or CHERIv9? It uses a lot of terms not defined in this spec like LG, GL, PERMIT_..., delegating, etc.

Could we reword this if we'd like to keep the general idea?

Reply:
Yeah this was taken from cheriot - I need to update this.